app/views/restaurants/show.html.erb

```diff
@@ -47,21 +47,24 @@
       <% @restaurant.reviews.order('created_at desc').each do |review| %>
         <div>
           <div>
-            <% if current_user == review.user %>
-              <div>
-                <%= link_to "Delete review", review_path(review), method: :delete %>
-                <%= link_to "Edit Review", edit_restaurant_review_path(review.restaurant, review) %>
-              </div>
-            <% end %>
+            <%# link_to "Up vote",  id: review.id %>
+            <%# link_to "Down vote", id: review.id %>
             <div>Rating(1-5):
```

Comment:
I would think you'd have div's for each little set of things so you can keep style separate:

```ruby
<div class="rating-title">Rating(1-5):</div>
<div class="rating">
  <%= review.rating %>
</div>
```